rollup.config.js

```diff
@@ -1,39 +1,59 @@
+import external from 'rollup-plugin-peer-deps-external';
+import json from '@rollup/plugin-json';
 import typescript from 'rollup-plugin-typescript2';
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import sourcemaps from 'rollup-plugin-sourcemaps';
+import { terser } from 'rollup-plugin-terser';
+import pkg from './package.json';
 
-export function getConfig({
-  tsconfig = './tsconfig.json',
-  output = [
-    {
-      file: `dist/resolvers.js`,
-      format: 'cjs',
-      exports: 'named',
-    },
-    {
-      file: `dist/resolvers.es.js`,
-      format: 'esm',
+export default [
+  {
+    input: 'src/index.ts',
+    output: {
+      name: 'HookFormResolvers',
```

Comment:
@bluebill1049 Is this umd name OK?
`HookFormResolvers` vs `ReactHookFormResolvers` vs `Resolvers`
please see [umd example](https://codesandbox.io/s/react-hook-form-umd-n8z43?file=/index.html).

Reply:
`ReactHookFormResolvers` 👍 what do you think? looks better at umd example.

Reply:
we may use `HookForm` as the umd name of react-hook-form.
Which is better?

* HookForm
* HookFormResolvers
* HookFormErrorMessage
* HookFormDevtools

vs

* ReactHookForm
* ReactHookFormResolvers
* ReactHookFormErrorMessage
* ReactHookFormDevtools

Reply:
yea better! first one is better!

Reply:
actually maybe second one, what if one day we have angularHookForm? lol

Reply:
Hahaha. if it has the potential to support Angular and Vue, I think `ReactHookForm` is better.
However, in that case we think it is necessary to reconsider the scope name (`@hookform`).
ex: `@hookform` to `@react-hook-form`

Reply:
@hookform/vue is fine as well?

if we can change to @hookform/react later on.

Reply:
awesome! I will change umd name from `HookForm` to `ReactHookForm`.

Reply:
done.